server/devdocs/index.js

```diff
@@ -211,8 +213,8 @@ module.exports = function() {
 	// return the content of a document in the given format (assumes that the document is in
 	// markdown format)
 	app.get( '/devdocs/service/content', ( request, response ) => {
-		let path = request.query.path;
-		const format = request.query.format || 'html';
+		let { path } = request.query;
+		const { format = 'html' } = request.query.format;
```

Comment:
Shouldn't this be

```suggestion
		const { format = 'html' } = request.query;
```

?

Reply:
My console seems to agree :grimacing: 

```
TypeError: Cannot destructure property `format` of 'undefined' or 'null'.
```

Reply:
Yes, I ruined the code when trying to migrate it to fancy ES6 destructuring! Fixed up in https://github.com/Automattic/wp-calypso/pull/38784/commits/04bdaec8b1032eb15c38e0e1ce2a208f92e933bf